cmd/power/power.go

```diff
@@ -89,19 +138,7 @@ func runDocker(name string) error {
 		})
 	}()
 
-	err = client.StartContainer(powerContainer.ID, powerContainer.HostConfig)
-	if err != nil {
-		return err
-	}
-
-	_, err = client.WaitContainer(powerContainer.ID)
```

Comment:
I'm taking a guess at why you did this, but I think it may help to point out that `shutdown -H <time>` runs in the foreground.  It will block until the shutdown time.  If you hit `ctrl+c` shutdown will be cancelled.